### PR TITLE
Ignore `.idea`

### PR DESCRIPTION
**Pull request by YangFong**
### Description

Git commits ignore `.idea`.

### Motivation

  
Configuration files are inevitable when editing documents using the JetBrains family of products. So much so that every time I need to prevent uploading these files.

### CODE REVIEW

**Review by Josh-Cena — APPROVED** [review]

**Review by caugner — APPROVED** [review]

**Comment by brent85** [comment]
> ### Description
> Git commits ignore `.idea`.
> 
> ### Motivation
> Configuration files are inevitable when editing documents using the JetBrains family of products. So much so that every time I need to prevent uploading these files.

YangFong:git-config

**Review by brent85 — COMMENTED** [review]
.gitignore

File: .gitignore
Comment (brent85) [inline .gitignore:1]: https://github.com/mdn/content/pull/20879#issue-1377439462

**Comment by brent85** [comment]
> Contributor

https://github.com/brent85/content.git

**Comment by Ah80184994** [comment]
636375ydtfeyfw

**Comment by Ah80184994** [comment]
9ue8yeggxgs5373

**Comment by Ah80184994** [comment]
https://www.facebook.com/share/v/19QECFJV6W/?mibextid=xfxF2i

**Pull request merged**